src/utils.js

```diff
@@ -0,0 +1,54 @@
+import XLSX from "xlsx";
+import { saveAs } from 'file-saver';
+
+const createExcelFile = (data) => {
+    let wb = XLSX.utils.book_new();
+    wb.SheetNames.push("Свод");
+
+    const ws_header = ['Страховой номер','Фамилия','Имя','Отчество','Дата увольнения','Дата приема - перевода'];
+    const ws_data_set = data.flat().map((person) => Object.keys(person).map(key => ([
+        [person[key].value]])));
+
+    wb.Sheets["Свод"] = XLSX.utils.aoa_to_sheet([ws_header, ...ws_data_set]);
+
+    return XLSX.write(wb, {bookType:'xlsx',  type: 'binary'});
+  } 
+
+const createExcelFile2 = (data) => {
+    let wb = XLSX.utils.book_new();
+
+    wb.SheetNames.push("Свод");
+    const ws_header = ['Страховой номер','Фамилия','Имя','Отчество','Дата увольнения','Дата приема - перевода'];
+    const ws_data_set = data.flat().map((person) => Object.keys(person).map(key => ([
+      [person[key].value]])));
+
+    wb.Sheets["Свод"] = XLSX.utils.aoa_to_sheet([ws_header, ...ws_data_set]);
+
+    data.forEach((el, index) => {
```

Comment:
move to separate function (think about pure if posible)

Reply:
not done, BUT
Instead I created one function called "createExcelFile"  that creates "Свод" sheet and additional sheets if needed
Need to discuss in case you don't like it :)